src/app/components/EmailInput/EmailInput.module.css

```diff
@@ -0,0 +1,68 @@
+.emailInput {
+    height: fit-content;
+    display: flex;
+    flex-direction: column;
+    margin: 8px;
+    padding-top: 8px;
+    width: 25ch;
+    position: relative;
+}
+
+.emailInputContainer {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    width: 100%;
+    font-weight: 400;
+    font-size: 1rem;
+    line-height: 1.4375em;
+    letter-spacing: 0.00938em;
+    color: white;
+    cursor: text;
+    border-radius: 4px;
+}
+
+.emailInputLabel,
+.emailInputHelperText,
+.emailInputErrorText {
+    font-size: 0.75rem;
```

Comment:
it should be around 12px. In order to make it work, the root should be 16px. For now to make it simple, lets just use 12px here.
<img width="418" alt="image" src="https://github.com/user-attachments/assets/cd4da4cd-7b39-4822-9e40-b1ae81913a19" />

Reply:
Got it, changed it to 12px.